DATA/production/qc-workflow.sh

```diff
@@ -259,6 +262,12 @@ elif [[ -z ${QC_JSON_FROM_OUTSIDE:-} ]]; then
     fi
   done
 
+  # CTF QC
+  if [[ ! -z "${QC_JSON_CTF_SIZE:-}" ]]; then
+    add_QC_JSON GLO_CTF ${QC_JSON_CTF_SIZE}
+    add_pipe_separated QC_DETECTOR_CONFIG_OVERRIDE '.qc.tasks.CTFSize.taskParameters.detectors=${WORKFLOW_DETECTORS}'
```

Comment:
probably doesn't matter, but shouldn't this be WORKFLOW_DETECTORS_CTF?

Reply:
all I want to achieve is that plots for one detector are not published by different runs, so WORKFLOW_DETECTORS is ok. In case for one detector CTF storage is disabled then the plots would show zero size which is fine I think.